docs/settings/reporting-settings.asciidoc

```diff
@@ -104,15 +104,14 @@ security is enabled, `xpack.security.encryptionKey`.
 [cols="2*<"]
 |===
 | `xpack.reporting.queue.pollInterval`
-  | Specifies the number of milliseconds that the reporting poller waits between polling the
-  index for any pending Reporting jobs. Defaults to `3000` (3 seconds).
+  | Specify the {ref}/common-options.html#time-units[time] that the reporting poller waits between polling the index for any
```

Comment:
Might be nice to give a brief description on the order of these timeouts, so folks know what to expect with regards to the lifecycle of how these timeouts.